README.md

```diff
@@ -293,8 +309,27 @@ If you are looking for something that isn't yet covered by this documentation yo
 
 ### Laravel
 
-In Laravel you need to create a GraphQL service provider, a GraphQL service, a GraphQL controller and map a route for
-handling the GraphQL requests.
+Here is a example that demonstrates how you can use this library in your Laravel project. You need an application 
+service to expose this library to your application, a service provider to register that service, a controller and a 
+route for handling the GraphQL POST requests.
```

Comment:
"`POST` requests"

Google suggest using code font for `POST`: https://developers.google.com/style/code-in-text